src/test/rgw/bucket_notification/README.rst

```diff
@@ -0,0 +1,43 @@
+============================
+ Bucket Notification tests
+============================
+
+The tests use the Nose test framework. To get started, ensure you have
+the ``virtualenv`` software installed; e.g. on Debian/Ubuntu::
+
+        sudo apt-get install python-virtualenv
+
+and then run::
+
+        ./bootstrap
+
+You will need to use the sample configuration file named ``bntests.conf.SAMPLE`` 
+that has been provided at ceph/src/test/rgw/bucket_notification. This file can be
```

Comment:
please put directory names as formatted text. e.g. `/directory/`
also, be consistent with `/path/to/ceph/src/...`